src/jit/compiler.h

```diff
@@ -5710,6 +5711,7 @@ class Compiler
                 __int64 lconVal;
                 double  dconVal;
                 Range   u2;
+                bool    edge;
```

Comment:
Well, this edge stuff is definitely wrong as it fails to solve the problem it's supposed to solve - avoid creating unnecessary assertions. If the edge information is included in the assertion then it must be considered by the `Equals` check so we risk creating up to 3 assertions for the same `vnIdx` `vnLen` pair.

We need to somehow pass this information from `optCreateJTrueBoundsAssertion` to `optComputeAssertionGen` but it has to be "out of band", probably in a GTF flag or in a bit taken from `AssertionIndex`.